File: ember/app/models/place.js
import DS from 'ember-data';
import { attr, belongsTo } from '@ember-decorators/data';


export default class extends DS.Model {
  @belongsTo('rpa', {async: true}) rpa
  @belongsTo('county', {async: true}) county

  @attr('string') namelsad
  @attr('boolean' , { default: true }) ispublic
  //@attr('json') geojson

}

File: rails/app/models/place.rb
class Place < ApplicationRecord
  self.primary_key = 'id'
  belongs_to :county
  belongs_to :rpa
  include PgSearch::Model
  include ActiveModel::Dirty
  pg_search_scope :search_by_namelsad, against: [:namelsad], using: { tsearch: { any_word: true } }
  

 
end
